pkg/rpc/client/nep5.go

```diff
@@ -172,7 +173,11 @@ func (c *Client) CreateTxFromScript(script []byte, acc *wallet.Account, sysFee,
 		sysFee = result.GasConsumed
 	}
 
-	tx := transaction.New(c.opts.Network, script, sysFee)
+	network := c.GetNetwork()
+	if network == 0 {
```

Comment:
It can be valid network number.

Reply:
Added `initDone` bool flag to the client. It is set to `true` after `(Client).Init()` is done.